config/config.go

```diff
@@ -215,6 +219,16 @@ func NewConfig() (*Config, []cli.Flag) {
 			EnvVar:      "SPECTATORD_SERVICE_IMAGE",
 			Destination: &cfg.SpectatordServiceImage,
 		},
+		cli.BoolFlag{
+			Name:        "container-atlasd",
+			EnvVar:      "CONTAINER_ATLASD",
+			Destination: &cfg.ContainerAtlasd,
+		},
+		cli.StringFlag{
```

Comment:
Do we need both?

Why can't it just be the "image" flag, and if `""` we don't run it?

Reply:
@solarkennedy and I decided to take this up in a separate pull request.